src/SFA.DAS.FindEmploymentSchemes.UnitTests/Program.cs

```diff
@@ -0,0 +1,24 @@
+﻿
```

Comment:
Is this file needed?

Reply:
Yes, it's left over from integration tests - I've removed it